### ISSUE TITLE
🐛 BUG: "less than" breaks ternary expressions

### ISSUE THREAD

**Issue by WaltzingPenguin** [snowpackjs/astro#1241]
### What package manager are you using?

npm

### What operating system are you using?

Windows

### Describe the Bug

Creating an Astro component containing
```
<p>{ (3 < 4) ? 'true' : 'false' }</p>
```

results in the error
```
[executing astro] CompileError [ParseError]: Expected =
```

### Steps to Reproduce

See above

**Comment by natemoo-re** [comment]
Thanks for the detailed reproduction, @coelmay! These kinds of issues are exactly why we rewrote our compiler.

The new compiler and build engine for Astro is in preview under `astro@next` and reflected on our `main` branch. I just confirmed that [all of these cases](https://stackblitz.com/edit/github-vjtxnq?file=src%2Fpages%2Findex.astro) are working in that release.

Leaving this open until that is released as stable!

**Comment by natemoo-re** [comment]
Following up on my previous comment, the team has decided to close out issues that have been confirmed as fixed by [astro@0.21.0-next.0](https://github.com/snowpackjs/astro/releases/tag/astro%400.21.0-next.0), [astro@0.21.0-next.1](https://github.com/snowpackjs/astro/releases/tag/astro%400.21.0-next.1), or [astro@0.21.0-next.2](https://github.com/snowpackjs/astro/releases/tag/astro%400.21.0-next.2). Our hope is that this will help the [v0.21 milestone](https://github.com/snowpackjs/astro/milestone/1) remain as actionable as possible.

To verify that this issue has been fixed, you may
- Open a new [`astro@next` project on StackBlitz](https://astro.new/minimal@next)
- Locally upgrade to the `astro@next` preview
    ```shell
    npm i astro@next
    # If using Framework renderers...
    npm i @astrojs/renderer-name@next
    ```

If you think we made a mistake, **please reopen this issue**!

If you run into any other problems with `astro@next`, please [open a new issue](https://github.com/snowpackjs/astro/issues/new?assignees=&labels=0.21&template=---01-bug-report.yml&title=%F0%9F%90%9B+BUG%3Aastro@next).
